packages/camera/camera_android_camerax/example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.8.0'
+    ext.kotlin_version = '1.7.10'
```

Comment:
Why does this have version 1.7.10 but the code here packages/camera/camera_android_camerax/android/build.gradle 
Has 1.8.10?

Reply:
We explicitly want to test that the plugin works out of the box for users with a default `flutter create` setup, which still uses 1.7.10. Otherwise anyone using this plugin gets confusing compile errors and has to change their app settings.